wordpress-mobile/WordPress-iOS
# Issue/9856 make active plan optional

Fixes #9856 

This PR makes it optional for the user's currently active plan to be one of the plans returned from the `v1.2/sites/site/plans` endpoint.  This allows the plans feature to continue showing the four default plans instead of an error message prompting the user to contact support.  This matches the behavior in the wpandroid.

To test:
You want a site with a Jetpack or two year plan.  DM me and I help get you set up.
View MySites > Site > Plans and confirm that you see the four default plans.

Waiting for an update to the WordPressKit pod.  @ctarda and I both have PRs to bump to 1.4.1-beta.2 so once his lands I'll update this branch and update/remove the status labels.

After the patch:
![simulator screen shot - iphone 8 - 2018-10-04 at 11 46 19](https://user-images.githubusercontent.com/1435271/46486193-3bc73680-c7cb-11e8-8ee8-a4c064fd8b1d.png)
